test/helpers/helpers.js

```
@@ -19,6 +19,27 @@ class Helpers {
     }
   }
 
+  registration(){
+      browser.url('/');
+      $('#registration').click();
```

You use hardcoded value here. If it changes tomorrow, everything will fail. Think how to use a value from selectors.

---

@artsenius Do I need to create selector file .js for helpers?

---

and add to it selector for #registration?

---

there asre two ways:
- import existing selector into helpers.js
- re-write function to be able to accept arguments and pass this argument where you call it.
However, I think the first is better.

---

fixed